### PR TITLE
add Filecoin task

### PR DESCRIPTION
**Pull request by liberhe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ DAPP架构请参考文章--[从架构维度看Web2.0与Web3.0应用之别](https
 33. [pooltogether](basic/33-pooltogether/README.md) ⌛
 34. [subgraph](basic/34-subgraph/readme.md) ⬜
 35. [Liquity](https://github.com/liquity) ⬜
-36. [NFT IPFS](basic/36-nft-ipfs/README.md) ✅
+36. [FileCoin](basic/36-nft-filecoin/README.md) ✅
```
Comment (xrdavies) [inline README.md:123]: basic/36 is nothing related to filecoin. Why  rename it again to filecoin?

**Review by xrdavies — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (liberhe) [reply README.md:123]: task 10 has already finish nft-ipfs function.

Comment (xrdavies) [reply README.md:123]: So it's duplicate one, right?

Comment (xrdavies) [reply README.md:123]: So let's remove this one or replace it with another real filecoin task? @liberhe

Comment (liberhe) [reply README.md:123]: yes. This task is aim to research Filecoin.  Just replace it.

Comment (xrdavies) [reply README.md:123]: > yes. This task is aim to research Filecoin. Just replace it.

Could you add a TODO for this task?

Comment (liberhe) [reply README.md:123]: o k

**Review by yingjingyang — APPROVED** [review]

**Pull request closed without merging**